src/store/metadata/stub/content/editions/versions/public-funded-gross_2023-10.json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/public-funded-gross-expenditure/editions/2023-10/versions",
+    "@type": "hydra:Collection",
+    "title": "Public funded gross for expenditure on research and development",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/public-funded-gross-expenditure/editions/2023-10/versions/1",
+            "identifier": "1",
+            "issued": "2023-10-27T10:41:11Z",
+            "title": "Public funded gross for expenditure on research and development",
+            "summary": "Total UK public-funded expenditure on research and development for financial year 2020 - 2021.",
+            "description": "This dataset shows the total UK public-funded gross regional capital and non-capital expenditure on research and development (experimental statistics), in the UK, for financial year 2020 - 2021.\n\n### Research and development refers to any creative and systematic activity that contains certain amount of novelty – which includes the development of new concepts, products and processes – with the aim of increasing the stock of knowledge.",
+            "download_url": "https://staging.idpd.uk/datasets/public-funded-gross-expenditure/editions/2023-10/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                        "name": "period",
+                        "datatype": "string",
+                        "titles": "Year",
+                        "description": "The time period associated with the observation."
+                    },
+                    {
+                        "name": "areanm",
+                        "datatype": "string",
+                        "titles": "Area Name",
+                        "description": "The name associated with the geography code."
+                    },
+                    {
+                        "name": "areacd",
+                        "datatype": "string",
+                        "titles": "ONS Geography Code",
+                        "description": "The geography code associated with the observation."
+                    },
+                    {
+                        "name": "geography",
+                        "datatype": "string",
+                        "titles": "Geography Type",
+                        "description": "The categorisation of the geography code."
+                    },
+                    {
+                        "name": "observation",
+                        "datatype": "decimal",
+                        "titles": "Gross disposable household income per head",
```

Comment:
The observation refers to gross disposable income, but the dataset is for gross for expenditure on R&D